chapters/equations.tex

```diff
@@ -200,6 +200,7 @@ \subsection{When-Equations}\label{when-equations}
 \end{lstlisting}
 
 The \lstinline!expression! of a \lstinline!when!-equation shall be a discrete-time \lstinline!Boolean! scalar or vector expression.
+If \lstinline!expression! is a clocked expression it is handled differently, see \cref{clocked-when-clause}.
```

Comment:
Also make terminology crystal clear:
```suggestion
If \lstinline!expression! is a clocked expression, the equation is referred to as a \emph{clocked \lstinline!when!-clause} (\cref{clocked-when-clause}) rather than a \lstinline!when!-equation, and is handled differently.
```
(Moved the cross reference just to reduce the large number of commas by one.)